config/crd/bases/v1beta1/datadoghq.com_watermarkpodautoscalers.yaml

```diff
@@ -305,6 +311,7 @@ spec:
               minimum: 1
               type: integer
           required:
+          - minAvailableReplicaPerc
```

Comment:
Is making a new field mandatory considered as a breaking change ?
(I.e.: currently existing WPA don’t have a `minAvailableReplicaPerc` setting)

Reply:
Is there a default value for `minAvailableReplicaPerc` if it’s not set ?
If it’s 0, there’s a risk of blocking all WPA as soon as a single pod land on an unhealthy node.